src/uct/base/uct_cm.c

```diff
@@ -60,15 +60,46 @@ ucs_status_t uct_listener_reject(uct_listener_h listener,
     return listener->cm->ops->listener_reject(listener, conn_request);
 }
 
-UCS_CLASS_INIT_FUNC(uct_cm_t, uct_cm_ops_t* ops, uct_component_h component)
+
+#if ENABLE_STATS
+static ucs_stats_class_t uct_cm_stats_class = {
+    .name           = "",
+    .num_counters   = 0
+};
+#endif
+
+UCS_CLASS_INIT_FUNC(uct_cm_t, uct_cm_ops_t* ops, uct_iface_ops_t* iface_ops,
+                    uct_component_h component)
 {
-    self->ops       = ops;
-    self->component = component;
-    return UCS_OK;
+    self->ops                     = ops;
+    self->component               = component;
+    self->iface.super.ops         = *iface_ops;
+
+    self->iface.worker            = NULL;
```

Comment:
why worker is NULL? we have worker when opening CM

Reply:
but this is a dummy iface, the worker field through it shouldn't be accessed.

Reply:
maybe we can save the worker here and remove the field from cm?

Reply:
where would we get the worker from then?

Reply:
add another param to uct base cm c'tor